tardis/transport/montecarlo/base.py

```diff
@@ -114,9 +115,13 @@ def initialize_transport_state(
         )
 
         geometry_state = simulation_state.geometry.to_numba()
-
-        opacity_state_numba = opacity_state_to_numba(
-            opacity_state, macro_atom_state, self.line_interaction_type
+        opacity_state_numba = OpacityState.from_plasma(
```

Comment:
An opacity state is already being passed to this function

Reply:
https://github.com/tardis-sn/tardis/pull/2932#discussion_r1924265207 and also there is a merge conflict.